network/v1/types.go

```diff
@@ -54,33 +73,55 @@ type ClusterNetworkList struct {
 	Items []ClusterNetwork `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
+// +kubebuilder:validation:Pattern=`^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$`
+// HostSubnetEgressIP represents one egress IP address currently hosted on the node represented by
+// HostSubnet
+type HostSubnetEgressIP string
```

Comment:
This changes the API in a way that will require updates to openshift/sdn. If you're only doing it to avoid having to specify the validation pattern in two places you should just revert that

Reply:
the generated CRD is still showing the items slice as made of `string` type. i compiled the openshift/sdn using the newly generated crds and didn't see any build errors.

Reply:
no, I mean you're changing the *golang* API. With this change sdn would need to cast between `string` and `HostSubnetEgressIP` in some places once it vendored the latest openshift/api.